lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -71,7 +107,11 @@ public static int computeFactorial(int n) {
    * @return The sum of all the values.
    */
   public static double addNumbers(double[] values) {
-    return 0;
+    double sum = 0;
```

Comment:
You can do this is one line of code using Stream.reduce. See if you can make that work.

Reply:
Cool! Reduced to one line!